src/assets/user-info-settings/index.css

```
@@ -0,0 +1,255 @@
+html {
+  background: red;
+}
+
+body {
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  font-size: 15px;
+  width: 1280px;
+  margin: 0;
+  overflow: auto;
+  box-sizing: border-box;
+  background: #eaeaea;
+}
+button:active, .server-selection input:checked + label {
+  box-shadow: inset 0 0px 10px 0 rgba(66,45,120,.75) !important;
```

For readability purposes

```suggestion
  box-shadow: inset 0 0px 10px 0 rgba(66, 45, 120, 0.75) !important;
```

These kinds of spacing issues are seen multiple times here, but also only commenting once